apollo-core/src/main/java/com/ctrip/framework/apollo/core/ApolloClientSystemConsts.java

```diff
@@ -146,4 +146,14 @@
    * enable property names cache environment variables
    */
   public static final String APOLLO_PROPERTY_NAMES_CACHE_ENABLE_ENVIRONMENT_VARIABLES = "APOLLO_PROPERTY_NAMES_CACHE_ENABLE";
+
+  /**
+   * enable property names cache
+   */
+  public static final String APOLLO_PROPERTY_FILE_CACHE_ENABLE = "apollo.property.file.cache.enable";
+
+  /**
+   * enable property names cache environment variables
+   */
+  public static final String APOLLO_PROPERTY_FILE_CACHE_ENABLE_ENVIRONMENT_VARIABLES = "APOLLO_PROPERTY_FILE_CACHE_ENABLE";
```

Comment:
How about simply apollo.file.cache.enable?
```suggestion

  /**
   * enable property names cache
   */
  public static final String APOLLO_FILE_CACHE_ENABLE = "apollo.file.cache.enable";

  /**
   * enable property names cache environment variables
   */
  public static final String APOLLO_FILE_CACHE_ENABLE_ENVIRONMENT_VARIABLES = "APOLLO_FILE_CACHE_ENABLE";
```

Reply:
I rebase the code, fixed